test-network/CHAINCODE_AS_A_SERVICE_TUTORIAL.md

```diff
@@ -71,12 +71,14 @@ peer chaincode query -C mychannel -n basicts -c '{"Args":["org.hyperledger.fabri
 
 If you don't have `jq` installed omit `| jq`.  The metadata shows the details of the deployed contract and is JSON, so jq makes it easier to read.  You can repeat the above commands for org2 to confirm that is working.
 
-To run the Java example, change the `deployCCAAS` command as follows, This will create two new containers.
+To run the Java example, change the `deployCCAAS` command as follows. This will create two new containers.
 
 ```bash
 ./network.sh deployCCAAS  -ccn basicj -ccp ../asset-transfer-basic/chaincode-java
 ```
 
+Note that all the asset-transfer-basic application samples use 'basic' as the chaincode name and need to be adjusted to use the name 'basicts' or 'basicj' accordingly, or you need to use the name 'basic' in the commands above.
```

Comment:
I found this sentence a bit confusing, and have similar feedback to the comment above.